### DISCUSSION TITLE
How to show the exact numerical value of unit strength by using Phobos?

### DISCUSSION THREAD

**Discussion by 8abs** [Phobos-developers/Phobos#607] [Q&A] [answered]
I saw this feature in this video, they told me I need to download a phobos to activate this function. But I can't find how to show the value of units in the manual. Is there anyone can tell me ? I would be so pleased and grateful.
https://www.bilibili.com/video/BV17541117v7#reply110444242112

**Comment by MortonPL** [comment, marked as answer]
This feature is currently under development (see #589) and isn't in the latest public build yet. However, you can use it by going to the linked page and downloading the test build there.
If you encounter any issues with it, please comment on the linked page.
